README.md

```diff
@@ -65,6 +65,12 @@ These macros perform *compile-time* alignment and size checks, but cannot be
 used in generic contexts. For generic conversions, use the methods defined
 by the [conversion traits](#conversion-traits).
 
+###### Dynamically-sized types
+
+Zerocopy supports sized types, slices, and [slice DSTs][slice-dsts].
+
+[slice-dsts]: KnownLayout#dynamically-sized-types
```

Comment:
IMO, this sentence should just be moved under the "Conversion Traits" heading, rather than introducing a new heading.

Reply:
Done.